osu.Game/Online/Chat/UserChat.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Game.Users;
+
+namespace osu.Game.Online.Chat
+{
+    public class UserChat : ChatBase

```

Comment:
Why can't this just derive from `Channel` (and be called `PrivateChannel`)?

Reply:
I didn't do that because then would userchannel be polluted with stuff it doesn't need. I named it userchat, because the player chats with another user or in a channel. (And user... because Chatbase.Targettype.User) 
Private channel sound more like a chat where more than 2 users are present. Like a private chatroom, or groupchat.

Reply:
Everything should be sharing a common `Channel` interface, because that is how they are server-side. None of the existing properties are polluting because they will be available in private chats too.